src/server/helper.ts

```diff
@@ -116,6 +116,29 @@ class Helper {
       return '';
     return '\n' + '='.repeat(20) + ' Browser output: ' + '='.repeat(20) + '\n' + logs.join('\n');
   }
+
+  static parseRemoteAddress(value?: string) {
+    if (!value)
+      return;
+
+    const colon = value.lastIndexOf(':');
+    const dot = value.lastIndexOf('.');
+    try {
+      if (colon > dot) { // IPv4
+        const [address, port] = value.split(':');
+        return {
+          ipAddress: address,
+          port: +port,
+        };
+      } else { // IPv6
+        const [address, port] = value.split('.');
+        return {
+          ipAddress: `[${address}]`,
+          port: +port,
+        };
+      }
+    } catch (_) {}
+  }
```

Comment:
Is this too naïve? FF and CR give us the info split up with the proper typing, WK we have to parse it out ourselves.